src/main/java/eu/delta_source/internship/Cart.java

```diff
@@ -0,0 +1,91 @@
+package eu.delta_source.internship;
+
+import java.util.ArrayList;
+
+/**
+ * The Cart class is used to add delivery fee and VAT to the items from the Cart.
+ *
+ * @author Evgeni Georgiev
+ * @version 1.0
+ */
+public class Cart {
+	private final ArrayList<CartItem> cartItems = new ArrayList<>();
+	private double deliveryFee;
+
+	public double getDeliveryFee() {
+		return this.deliveryFee;
+	}
+
+	// Method outputting details for single product(label, price, quantity, result sum for price and quantity).
+	public void runProductsInCartOutput() {
```

Comment:
Description is good.
Method name is not. Think of something describing what the method does.